doc/getting-started.md

```diff
@@ -52,7 +52,7 @@ This command will start the APISIX server and the admin API will be available in
 
 ```bash
 $ git clone https://github.com/apache/incubator-apisix-docker.git
-$ cd example
+$ cd incubator-apisix-docker/example
```

Comment:
https://github.com/apache/apisix-docker

the latest address should be this one. please update it